PythonVisualizations/Array.py

```diff
@@ -332,3 +353,28 @@ def clickDelete(self):
 - make it look pretty
 - animate insert
 '''
+
+''''
+Tested:                             Expected Result:                 Result: 
+- Insert number in range            - Inserted into array            - Inserted into array
+- Insert number out of range        - Error message                  - Error message
+- Insert non-int *                  - Unable to type character       - Unable to type character
+- Insert when full                  - Error message                  - Error message
+- Find number in Array              - Number found                   - Number found
+- Find non-int *                    - Unable to type character       - Unable to type character
+- Find number in Array              - Number found                   - Number found
+  with 0 before (07 = 7)
+- Find number not in Array          - Number searched for            - Number searched for        
+                                      and declared not in array        and declared not in array
+- Find first of duplicate numbers   - Number searched for and found  - Number searched for and found
+- Delete a number in Array          - Number deleted from array      - Number deleted from array
+- Delete a number not in Array      - Error message                  - Error message
+- Delete first of duplicate numbers - First duplicate deleted        - First duplicate deleted
+- Delete from empty Array           - Error message                  - Error message
+- Delete non-int *                  - Unable to type character       - Unable to type character
+- Delete right-most                 - Right most number deleted      - Right most number deleted
+- Delete right-most (Array empty)   - Error message                  - Error message
+- Speed switch works                - Animation is smooth and in     - Animation is smooth and in
+                                      tune with set speed              tune with set speed
+*(symbols and letters)
+'''
```

Comment:
The test plan and test results should go in another file, not in the source code itself.  I believe that's what was done in PR #22. 
Also the comments in the ToDo list above it belong in the Issue tracking database.